src/api/aux/utils.ts

```diff
@@ -14,7 +14,10 @@ export function blobToByteArray(blob: Blob): Promise<Uint8Array | null> {
       }
 
       if (typeof reader.result === 'string') {
-        const b64 = reader.result.split('data:application/octet-stream;base64,')[1]
+        const b64 = reader.result.split('base64,')[1]
+        if (!b64) {
```

Comment:
Could I kindly ask you to add an empty line before this line (`if (!b64) {`), to make the code slightly more readable, please?